app/views/layouts/admin.html.erb

```diff
@@ -71,6 +71,14 @@
         </div>
       <% end %>
       <%= yield %>
+      <% if @help_url %>
+        <a class="admin-help-button crayons-btn crayons-btn--icon-rounded crayons-btn--s" href="<%= @help_url %>" target="_blank">
+          <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="54px" height="54px">
+            <path fill="none" d="M0 0h24v24H0z"/>
+            <path fill="#ffffff" d="M12 22C6.477 22 2 17.523 2 12S6.477 2 12 2s10 4.477 10 10-4.477 10-10 10zm0-2a8 8 0 1 0 0-16 8 8 0 0 0 0 16zm-1-5h2v2h-2v-2zm2-1.645V14h-2v-1.5a1 1 0 0 1 1-1 1.5 1.5 0 1 0-1.471-1.794l-1.962-.393A3.501 3.501 0 1 1 13 13.355z"/>
+          </svg>
```

Comment:
can this be saved as a file? If we use one the `inline_svg_tag` we can also add the a11y label as it's missing here

Reply:
Made this change.